.github/workflows/deploy-to-development.yml

```diff
@@ -180,7 +180,8 @@ jobs:
 
   plan-production:
     environment: production
-
+    env:
+      TF_VAR_newrelic_license_key: ${{ secrets.NEW_RELIC_API_KEY }}
```

Comment:
Should we put this in SecretsManager and reference it from there?

Reply:
We already needed it in GH for the e2e test reporting